### ISSUE TITLE
Can't install on Debian

### ISSUE THREAD

**Issue by tarto-dev** [tableflip/guvnor#106]
I can't install this by using `$ npm i -g --unsafe-perm guvernor` but can get a log once I run it logged as root and using sudo (??) `root@debian:/ $ sudo npm i -g --unsafe-perm guvernor`.

I've installed npm by using : 
```bash
curl -sL https://deb.nodesource.com/setup_9.x | sudo -E bash -
apt-get install -y nodejs
npm i -g npm
```

```bash
root@debian:/# lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 9.5 (stretch)
Release:        9.5
Codename:       stretch
```

```bash
root@debian:/# node --version && npm --version
node v10.9.0
npm 6.4.1
```

**Comment by tarto-dev** [comment]
```bash
root@debian:/# sudo npm install -g guvnor --unsafe-perm
npm WARN deprecated bcrypt@0.8.7: bcrypt < v2.0.0 is susceptible to bcrypt wrap-around bug. Upgrade to bcrypt >= v2.0.0 for improved support for newer bcrypt hashes
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated hoek@2.14.0: The major version is no longer supported. Please update to 4.x or newer
npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
npm WARN deprecated hoek@2.16.3: The major version is no longer supported. Please update to 4.x or newer
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated hoek@0.9.1: The major version is no longer supported. Please update to 4.x or newer
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> v8-profiler@5.6.5 preinstall /usr/local/lib/node_modules/guvnor/node_modules/v8-profiler
> node -e 'process.exit(0)'


> v8-debug@0.7.7 preinstall /usr/local/lib/node_modules/guvnor/node_modules/v8-debug
> node -e 'process.exit(0)'

/usr/local/bin/bs -> /usr/local/lib/node_modules/guvnor/bin/guv
/usr/local/bin/boss -> /usr/local/lib/node_modules/guvnor/bin/guv
/usr/local/bin/bs-web -> /usr/local/lib/node_modules/guvnor/bin/guv-web
/usr/local/bin/boss-web -> /usr/local/lib/node_modules/guvnor/bin/guv-web
/usr/local/bin/guv -> /usr/local/lib/node_modules/guvnor/bin/guv
/usr/local/bin/guv-web -> /usr/local/lib/node_modules/guvnor/bin/guv-web
/usr/local/bin/guvnor -> /usr/local/lib/node_modules/guvnor/bin/guv
/usr/local/bin/guvnor-web -> /usr/local/lib/node_modules/guvnor/bin/guv-web

> weak@1.0.1 install /usr/local/lib/node_modules/guvnor/node_modules/weak
> node-gyp rebuild

make: Entering directory '/usr/local/lib/node_modules/guvnor/node_modules/weak/build'
  CXX(target) Release/obj.target/weakref/src/weakref.o
In file included from ../../nan/nan.h:190:0,
                 from ../src/weakref.cc:18:
../../nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
../../nan/nan_maybe_43_inl.h:88:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
   return obj->ForceSet(GetCurrentContext(), key, value, attribs);
               ^~~~~~~~
In file included from ../src/weakref.cc:18:0:
../../nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’:
../../nan/nan.h:817:60: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, func, argc, argv);
                                                            ^
In file included from ../../nan/nan.h:47:0,
                 from ../src/weakref.cc:18:
/root/.node-gyp/10.9.0/include/node/node.h:172:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^
/root/.node-gyp/10.9.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/weakref.cc:18:0:
../../nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’:
../../nan/nan.h:831:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, symbol, argc, argv);
                                                              ^
In file included from ../../nan/nan.h:47:0,
                 from ../src/weakref.cc:18:
/root/.node-gyp/10.9.0/include/node/node.h:165:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^
/root/.node-gyp/10.9.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/weakref.cc:18:0:
../../nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’:
../../nan/nan.h:845:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, method, argc, argv);
                                                              ^
In file included from ../../nan/nan.h:47:0,
                 from ../src/weakref.cc:18:
/root/.node-gyp/10.9.0/include/node/node.h:158:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^
/root/.node-gyp/10.9.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/weakref.cc:18:0:
../../nan/nan.h: In member function ‘v8::Local<v8::Value> Nan::Callback::Call_(v8::Isolate*, v8::Local<v8::Object>, int, v8::Local<v8::Value>*) const’:
../../nan/nan.h:1463:5: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
     ));
     ^
In file included from ../../nan/nan.h:47:0,
                 from ../src/weakref.cc:18:
/root/.node-gyp/10.9.0/include/node/node.h:172:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^
/root/.node-gyp/10.9.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
weakref.target.mk:101: recipe for target 'Release/obj.target/weakref/src/weakref.o' failed
make: *** [Release/obj.target/weakref/src/weakref.o] Error 1
make: Leaving directory '/usr/local/lib/node_modules/guvnor/node_modules/weak/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:239:12)
gyp ERR! System Linux 4.9.0-8-amd64
gyp ERR! command "/opt/node/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/guvnor/node_modules/weak
gyp ERR! node -v v10.9.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok

> v8-debug@0.7.7 install /usr/local/lib/node_modules/guvnor/node_modules/v8-debug
> node-pre-gyp install --fallback-to-build

node-pre-gyp ERR! Tried to download(404): https://node-inspector.s3.amazonaws.com/debug/v0.7.7/node-v64-linux-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for v8-debug@0.7.7 and node@10.9.0 (node-v64 ABI, glibc) (falling back to source compile with node-gyp)
node-pre-gyp ERR! Tried to download(undefined): https://node-inspector.s3.amazonaws.com/debug/v0.7.7/node-v64-linux-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for v8-debug@0.7.7 and node@10.9.0 (node-v64 ABI, glibc) (falling back to source compile with node-gyp)
make: Entering directory '/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build'
  CXX(target) Release/obj.target/debug/src/debug.o
make: Entering directory '/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build'
  CXX(target) Release/obj.target/debug/src/debug.o
../src/debug.cc:1:22: fatal error: v8-debug.h: No such file or directory
 #include <v8-debug.h>
                      ^
compilation terminated.
debug.target.mk:104: recipe for target 'Release/obj.target/debug/src/debug.o' failed
make: *** [Release/obj.target/debug/src/debug.o] Error 1
make: Leaving directory '/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:239:12)
gyp ERR! System Linux 4.9.0-8-amd64
gyp ERR! command "/opt/node/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64/debug.node" "--module_name=debug" "--module_path=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64"
gyp ERR! cwd /usr/local/lib/node_modules/guvnor/node_modules/v8-debug
gyp ERR! node -v v10.9.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/opt/node/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64/debug.node --module_name=debug --module_path=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/guvnor/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:961:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:250:5)
node-pre-gyp ERR! ../src/debug.cc:1:22: fatal error: v8-debug.h: No such file or directory
 #include <v8-debug.h>
                      ^
Systemcompilation terminated.
 Linux 4.9.0-8-amd64
node-pre-gyp ERR! command "/opt/node/bin/node" "/usr/local/lib/node_modules/guvnor/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /usr/local/lib/node_modules/guvnor/node_modules/v8-debug
node-pre-gypdebug.target.mk:104: recipe for target 'Release/obj.target/debug/src/debug.o' failed
make: *** [Release/obj.target/debug/src/debug.o] Error 1
 make: Leaving directory '/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build'
ERR! node -v v10.9.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.39
node-pre-gyp ERR! not ok
gypFailed to execute '/opt/node/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64/debug.node --module_name=debug --module_path=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64' (1)
 ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:239:12)
gyp ERR! System Linux 4.9.0-8-amd64
gyp ERR! command "/opt/node/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64/debug.node" "--module_name=debug" "--module_path=/usr/local/lib/node_modules/guvnor/node_modules/v8-debug/build/debug/v0.7.7/node-v64-linux-x64"
gyp ERR! cwd /usr/local/lib/node_modules/guvnor/node_modules/v8-debug
gyp ERR! node -v v10.9.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/guvnor/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: weak@1.0.1 (node_modules/guvnor/node_modules/weak):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: weak@1.0.1 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! v8-debug@0.7.7 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the v8-debug@0.7.7 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-09-18T13_56_08_867Z-debug.log
```

**Comment by tarto-dev** [comment]
[lognpm.log](https://github.com/tableflip/guvnor/files/2392963/lognpm.log)
The logs

**Comment by elilambnz** [comment]
Solved in #105
